src/catalog/catalog.cpp

```diff
@@ -265,7 +282,8 @@ std::vector<std::string> Catalog::getMacroNames(transaction::Transaction* tx) co
 void Catalog::setTableComment(table_id_t tableID, const std::string& comment) {
     KU_ASSERT(readWriteVersion != nullptr);
     setToUpdated();
-    readWriteVersion->getTableSchema(tableID)->setComment(comment);
+    auto tableEntry = readWriteVersion->getTableCatalogEntry(tableID);
+    ku_dynamic_cast<CatalogEntry*, TableCatalogEntry*>(tableEntry)->setComment(comment);
```

Comment:
I think can associate comment to catalog entry which means we can comment any catalog entry.